### PR TITLE
Feature: Practice two

### PR DESCRIPTION
**Pull request by QuangTruong23**

### CODE REVIEW

**Comment by MinhNguyen41092** [comment]
All image files name should follow PascalCase, for example: `FacebookIcon.png`

**Comment by MinhNguyen41092** [comment]
You use too many duplicate images for responsive images. You need to use css background to control size and position of images.

**Comment by MinhNguyen41092** [comment]
You can use CSS to rotate the underline image, no need to use 2 of them.

File: practice-two/src/contact.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+    <!-- **** Fonts **** -->
+    <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&family=DM+Serif+Text&display=swap" rel="stylesheet">    <!-- **** Styles **** -->
+    <link rel="stylesheet" href="/src/styles/main.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
+<body>
+    <!-- **** Header **** -->
```
Comment (MinhNguyen41092) [inline practice-two/src/contact.html:14]: * Create pages folder.
* Move index.html and contact.html into pages folder.

File: practice-two/src/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <include src="../src/partials/social.html"></include>
+
+    </main>
+
+    <!-- **** Footer section **** -->
+
+        <include src="../src/partials/footer.html"></include>
+
+    <!-- **** HTML Inspector **** -->
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/html-inspector/0.8.2/html-inspector.js"></script>
```
Comment (MinhNguyen41092) [inline practice-two/src/index.html:44]: Use this in contact page too.

File: practice-two/src/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <title>UPBOX</title>
+    <!-- **** Fonts **** -->
+    <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&family=DM+Serif+Text&display=swap" rel="stylesheet">    <!-- **** Styles **** -->
+    <link rel="stylesheet" href="/src/styles/main.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
+<body>
+    <!-- Header -->
+
+        <include src="../src/partials/header.html"></include>
```
Comment (MinhNguyen41092) [inline practice-two/src/index.html:16]: Redundant indentation.

File: practice-two/src/partials/contact-header.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<header class="header">
+    <div class="topbar">
+    </div>
+    <div class="header__wrap display-none-mobile">
+        <h1 class="mb-0"><a href=""><img src="/src/assets/images/logo/Logo.png" class="logo-image" alt="Logo"></a></h1>
+        <nav class="header__navbar">
+            <div class="navbar__wrap">
+                <ul class="navbar__link mb-0">
+                    <li><a href="" class="navbar__link-item">MAKE UP</a></li>
```
Comment (MinhNguyen41092) [inline practice-two/src/partials/contact-header.html:9]: Duplicate HTML with below. Write html once and use CSS to control appearance.

File: practice-two/src/partials/contact-header.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+    <!-- Hamburger -->
+    <nav class="navbar p-3 navbar-light display-none-tablet-up">
+        <h1 class="m-0"><a href=""><img src="/src/assets/images/logo/Logo.png" class="navbar-brand p-0 m-0 align-self-center" alt="Logo"></a></h1>
+        <button class="navbar-toggler border-ouline-none display-none" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo03" aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
+        <img src="/src/assets/images/header/Hamburger.png"  alt="Hamburger button">
+        </button>
+        <div class="collapse navbar-collapse" id="navbarTogglerDemo03">
+          <ul class="navbar-nav mt-2">
+            <li><a href="" class="nav-item nav-link black-color">MAKE U</li>
```
Comment (MinhNguyen41092) [inline practice-two/src/partials/contact-header.html:28]: Use lowercase text and use CSS to make it uppercase.

File: practice-two/src/partials/header.html [hunk 6]
```diff
@@ -0,0 +1,38 @@
+<header class="header">
+    <div class="topbar">
+    </div>
+    <div class="header__wrap display-none-mobile">
+        <h1 class="mb-0"><a href=""><img src="/src/assets/images/logo/Logo.png" class="logo-image" alt="Logo"></a></h1>
+        <nav class="header__navbar">
+            <div class="navbar__wrap">
+                <ul class="navbar__link mb-0">
+                    <li><a href="" class="navbar__link-item">THIS MONTH</a></li>
```
Comment (MinhNguyen41092) [inline practice-two/src/partials/header.html:9]: Write HTML once.

File: practice-two/src/partials/letschat.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+                Let's Chat!
+            </h2>
+            <form action="">
+                <label class="contact__body-email-title">Email</label>
+                <input type="email" value="Enter your email" class="contact__body-email-input">
+                <label class="contact__body-message-title">Message</label>
+                <textarea name="" id="" cols="30" rows="5" class="contact__body-message-textarea">Enter your message here</textarea>
+                <label for="newletter" class="contact__body-checkbox-title">Opt-in to newsletter</label>
+                <input type="checkbox" value="Opt-in to newsletter" class="contact__body-checkbox-check" id="newletter">
+                <input type="submit" class="contact__body-submit" value="Submit">
```
Comment (MinhNguyen41092) [inline practice-two/src/partials/letschat.html:19]: Use button here.

File: practice-two/src/partials/letschat.html [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+            <img src="/src/assets/images/contact/purse.png" class="body-img" alt="illustration">
+            <img src="/src/assets/images/contact/purse-tablet.png" alt="illustration" class="body-img-tablet">
+            <img src="/src/assets/images/contact/purse-desktop.png" alt="illustration" class="body-img-desktop">
+        </figure>
+        <div class="contact__body-wrap">
+            <h2 class="contact__body-title">
+                Let's Chat!
+            </h2>
+            <form action="">
+                <label class="contact__body-email-title">Email</label>
```
Comment (MinhNguyen41092) [inline practice-two/src/partials/letschat.html:13]: Should wrap each label + input group into a div container to better control spacing between each group.

File: practice-two/src/partials/letschat.html [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+                <input type="email" value="Enter your email" class="contact__body-email-input">
```
Comment (MinhNguyen41092) [inline practice-two/src/partials/letschat.html:14]: Wrong use of value attribute here.

File: practice-two/src/partials/plan.html [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+        <h2 class="plan__title-text">
+            Pick your plan
+        </h2>
+        <figure class="title__line">
+            <img src="/src/assets/images/plan/underline.png" alt="Line">
+        </figure>
+    </div>
+    <div class="plan__card-wrap">
+        <div class="plan__card-one">
+            <h2 class="plan__card-expiry">
```
Comment (MinhNguyen41092) [inline practice-two/src/partials/plan.html:12]: Should not use h2 here.

File: practice-two/src/partials/works.html [hunk 11]
```diff
@@ -0,0 +1,64 @@
+<section class="works__section">
+    <div class="works__title">
+        <h2 class="works__title-text">
+            How it works?
+        </h2>
+        <figure class="works__underline">
+            <img src="/src/assets/images/works/underline.png" alt="Line">
+        </figure>
+    </div>
+    <div class="works__box-one">
```
Comment (MinhNguyen41092) [inline practice-two/src/partials/works.html:10]: Need better naming here. They are step of a process. Example: how-it-works__step-one

File: practice-two/src/styles/base/reset.css [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+	content: none;
+}
+
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+input, button, textarea{
+	border: 0;
```
Comment (MinhNguyen41092) [inline practice-two/src/styles/base/reset.css:58]: Indentation here is too large. Should be 2 or 4 spaces only.

File: practice-two/src/styles/base/reset.css [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+	outline: none;
+	box-sizing: border-box;
+}
+
+a:hover {
+	text-decoration: none !important;
```
Comment (MinhNguyen41092) [inline practice-two/src/styles/base/reset.css:64]: Why do you need !important here.

File: practice-two/src/styles/components/contact.css [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+    font-weight: 400;
+    font-family: var(--btn-font-family);
+    margin: 4px 0;
+}
+
+.contact__body-email-input,
+.contact__body-message-textarea {
+    padding: 10px;
+    width: 100%;
+    background-color: #f3f3f3;
```
Comment (MinhNguyen41092) [inline practice-two/src/styles/components/contact.css:44]: Too many places you are still using raw hex color. Should use variables.

File: practice-two/src/styles/components/contact.css [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+    }
+}
+
+@media only screen and (min-width: 768px) and (max-width: 991.98px){
+    .contact__body {
+        display: flex;
+        justify-content: space-around;
+    }
+
+    .body-img {
```
Comment (MinhNguyen41092) [inline practice-two/src/styles/components/contact.css:73]: .body-img are none for both tablet and desktop. Why do you need to define it twice?

File: practice-two/src/styles/components/header.css [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+    background-color: var(--white-color);
+    font-family: var(--btn-font-family);
+    font-weight: 500;
+    font-size: 10px;
+    letter-spacing: 1px;
+    cursor: pointer;
+}
+
+.border-ouline-none {
+    border: none !important;
```
Comment (MinhNguyen41092) [inline practice-two/src/styles/components/header.css:53]: Why !important here?

File: practice-two/src/styles/components/hero.css [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+
+.hero__title {
+    font-family: var(--title-font-family);
+    font-size: 48px;
+    font-weight: 400;
+    line-height: 56px;
+}
+
+.hero__description {
+    font-family: 'DM Sans', sans-serif;
```
Comment (MinhNguyen41092) [inline practice-two/src/styles/components/hero.css:13]: Use variable.

File: practice-two/src/styles/components/hero.css [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+    display: none;
+}
+
+.hero__art {
+    display: none;
+}
+
+@media only screen and (min-width: 768px) and (max-width: 991.8px){
+    .hero__section {
+        background-image: url(/src/assets/images/hero/BOX-tablet.png);
```
Comment (MinhNguyen41092) [inline practice-two/src/styles/components/hero.css:45]: Define variables for all images.

File: practice-two/src/styles/components/plan.css [hunk 19]
```diff
@@ -0,0 +1,177 @@
...
+.plan__description {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+    font-family: var(--btn-font-family);
+    padding-right: 45px;
+    margin-top: 14px;
+}
+
+@media only screen and (min-width: 375px) and (max-width: 767px) {
```
Comment (MinhNguyen41092) [inline practice-two/src/styles/components/plan.css:72]: It's not used?

File: practice-two/src/index.html (hunk 2 above)
Comment (QuangTruong23) [reply practice-two/src/index.html:44]: fixed

File: practice-two/src/index.html (hunk 3 above)
Comment (QuangTruong23) [reply practice-two/src/index.html:16]: fixed

File: practice-two/src/partials/letschat.html (hunk 7 above)
Comment (QuangTruong23) [reply practice-two/src/partials/letschat.html:19]: fixed

File: practice-two/src/partials/letschat.html (hunk 8 above)
Comment (QuangTruong23) [reply practice-two/src/partials/letschat.html:13]: fixed

File: practice-two/src/partials/letschat.html (hunk 9 above)
Comment (QuangTruong23) [reply practice-two/src/partials/letschat.html:14]: fixed

File: practice-two/src/partials/plan.html (hunk 10 above)
Comment (QuangTruong23) [reply practice-two/src/partials/plan.html:12]: fixed

File: practice-two/src/styles/components/hero.css (hunk 17 above)
Comment (QuangTruong23) [reply practice-two/src/styles/components/hero.css:13]: fixed

File: practice-two/src/styles/base/reset.css (hunk 12 above)
Comment (QuangTruong23) [reply practice-two/src/styles/base/reset.css:58]: fixed

File: practice-two/src/styles/components/contact.css (hunk 14 above)
Comment (QuangTruong23) [reply practice-two/src/styles/components/contact.css:44]: fixed

File: practice-two/src/styles/components/hero.css (hunk 18 above)
Comment (QuangTruong23) [reply practice-two/src/styles/components/hero.css:45]: fixed

File: practice-two/src/styles/components/header.css (hunk 16 above)
Comment (MinhNguyen41092) [reply practice-two/src/styles/components/header.css:55]: border-ouline-none there is a typo: should be border-outline-none

Comment (QuangTruong23) [reply practice-two/src/styles/components/header.css:55]: fixed

File: practice-two/src/partials/header.html (hunk 6 above)
Comment (QuangTruong23) [reply practice-two/src/partials/header.html]: fixed

File: practice-two/src/partials/contact-header.html (hunk 4 above)
Comment (QuangTruong23) [reply practice-two/src/partials/contact-header.html]: fixed

File: practice-two/src/styles/base/reset.css (hunk 13 above)
Comment (QuangTruong23) [reply practice-two/src/styles/base/reset.css:64]: to override the bootstrap property

**Pull request merged**